purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/PartialComponent.kt

```diff
@@ -0,0 +1,6 @@
+package com.revenuecat.purchases.paywalls.components
+
+/**
+ * Marker interface for partial components.
+ */
+internal sealed interface PartialComponent
```

Comment:
This used to be in `ComponentOverrides.kt`, but it has to remain in the `paywalls.components` package, where its implementing classes live (in `ImageComponent.kt` and `TextComponent.kt`).